app/utils/IdGenerator.scala

```diff
@@ -23,5 +23,7 @@ import javax.inject.{Inject, Singleton}
 @Singleton
 class IdGenerator @Inject()() {
 
+  def getUid: String = UUID.randomUUID().toString
```

Comment:
is this necicary? its just a copy of the generateCorrelationId def with a iffrent name.